### ISSUE TITLE
CLI doesn't get the arguments

### ISSUE THREAD

**Issue by jsomsanith-tlnd** [i18next/i18next-gettext-converter#113]
## 🐛 Bug Report

With i18next-conv@11.0.1, the cli doesn't work anymore, because it doesn't get the arguments.

## To Reproduce

```sh
i18next-conv -l en -s ./i18n/tui-components.json -t ./i18n/tui-components.po
```

produces

```sh
at least call with argument -l and -s.
(call program with argument -h for help.)
```

## Expected behavior

The command is correct, it should run the conversion process.

## Your Environment

Not linked to the env

**Comment by jsomsanith-tlnd** [comment]
I have done a fix PR: #112 112

**Comment by perrin4869** [comment]
Released in `11.0.2`

**Comment by jsomsanith-tlnd** [comment]
Thank you @perrin4869 for the release !
